syntax = "proto3";

import "review.proto";

message ReviewSet {
  repeated Review reviews = 1;
}
